apps/core/components/rangeDatePicker/RangeDatePickerTypes.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import * as React from 'react';
+
+export type Props = {|
+  +isVisible: boolean,
+  +mode?: 'date' | 'time' | 'datetime',
+  +datePickerMode?: 'calendar' | 'spinner' | 'default', // this prop is supported only on android
+  +date?: Date,
+  +minDate?: Date,
+  +maxDate?: Date,
+  +onConfirm: Date => void,
+  +onDismiss: () => void,
+  +labels: {|
+    +cancel: React.Node, // this prop is supported only on web & iOS
+    +confirm: React.Node, // this prop is supported only on iOS
```

Comment:
Comments for `cancel` and `confirm` are probably no longer valid. Buttons are currently visible on all platforms.

Reply:
Removed